lib/youtube-dl.js

```diff
@@ -11,7 +11,7 @@ var detailsPath = path.join(__dirname, '..', 'bin/details'), ytdlBinary;
 
 if (fs.existsSync(detailsPath)) {
   var details = JSON.parse(fs.readFileSync(detailsPath));
-  ytdlBinary = (details.path) ? details.path : path.resolve(__dirname, '..', 'bin', details.exec);
+  ytdlBinary = path.resolve(__dirname, '..', 'bin', details.exec);
```

Comment:
Don't rely on `.path` because this is a fixed path when you run the downloader, so the better way here is always point to the default folder, otherwise this code will break if you build something from your OS and distribute to other OS.